docs/docs/60-new-docs/50-user-guide/20-how-to-guides/16-working-with-warehouses.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Learn how to work effectively with Warehouses
+sidebar_label: Working with Warehouses
+---
+
+# Working with Warehouses
+
+Each Kargo warehouse is represented by a Kubernetes resource of type
+`Warehouse`.
```

Comment:
This is not direct feedback to you, as I think it was not part of your task. But I think it would help here if we had a brief introductory paragraph about what a "Warehouse" actually is. Something like:

```suggestion
A `Warehouse` is a fundamental Kargo resource that manages and tracks various artifact sources. It acts as a central hub for subscribing to and monitoring changes in:

- Container image repositories
- Git repositories
- Helm chart repositories

`Warehouse`s are responsible for watching these sources and producing new `Freight` whenever changes are detected in the subscribed repositories. Each piece of `Freight` represents a collection of specific versions of artifacts that can be promoted through your deployment pipeline.
```

Reply:
+1 to this suggestion. I'll add that a link to the Warehouse section of the core concepts doc couldn't hurt either, but...

Do _not_ replicate the description of Warehouses from the context doc here. It's too contextual. It doesn't work without a more in-depth explanation of Freight preceding it.

> If freight can be conceptualized as boxes, then warehouses are the places those boxes come from.
>
> In more technical terms, a warehouse monitors one or more repositories for new revisions of artifacts, and when it finds them, it packages the latest revision of each together as a new piece of freight that will enter the promotion pipeline.

So stick with the language @hiddeco's recommended and just work in link to the concepts doc.